snapcraft/plugins/ruby.py

```diff
@@ -0,0 +1,71 @@
+import os
+import platform
+import logging
+
+import snapcraft
+from snapcraft import sources
+
+
+logger = logging.getLogger(__name__)
+
+
+class RubyPlugin(snapcraft.BasePlugin):
+  @classmethod
+  def schema(cls):
+    schema = super().schema()
+
+    schema['properties']['gems'] = {
+      'type': 'array',
+      'minitems': 1,
+      'uniqueItems': True,
+      'items': {
+        'type': 'string',
+      },
+      'default': [],
+    }
+    schema['properties']['will-install-gems'] = {
```

Comment:
I don't really understand what this option does. It has "gems" in the name, but really just seems to optionally add build packages to stage packages. What does that have to do with gems?